src/foam/u2/view/TitledArrayView.js

```diff
@@ -113,22 +126,30 @@ foam.CLASS({
           this
             .forEach(data || [], function(e, i) {
               var row = self.CollapsableRow.create({ index: i, value: e, collapsed: self.collapseBehaviour == 'START_COLLAPSED' ? true : false });
+              var summaryType = self.title || row.value.toSummary ? row.value$.map(v => v.toSummary()) : 'default';
```

Comment:
Why default? Because you're using it as a CSS class? But you're still displaying the value. I don't think it's appropriate to use the same value for both since I would like toSummary() to  potentially contain a lot of information, like type, name, help text, etc.
If you want to have a custom CSS class then check if the row has a .toCSSClassName() method or something.